py3status/modules/vpn_status.py

```diff
@@ -125,8 +125,11 @@ def return_status(self):
                 name = vpn
                 color = self.py3.COLOR_GOOD
 
-        # Format and create the response dict
-        full_text = self.py3.safe_format(self.format, {'name': name})
+        if name is None:
+            full_text = self.py3.safe_format(self.format)
+        else:
+            full_text = self.py3.safe_format(self.format, {'name': name})
+
```

Comment:
This change is not needed.  Because `{'name': None}` will cause the desired effects so we can just keep things simple.  Other than this I think it is a cool improvement to the module